ovh/resource_me_api_oauth2_client_test.go

```diff
@@ -0,0 +1,104 @@
+package ovh
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+// Returns an error if a string is empty,
+// Used in the calls to resource.TestCheckResourceAttrWith() below
+func apiOauth2ClientStringNotEmpty(s string) error {
+	if len(s) > 0 {
+		return nil
+	}
+	return fmt.Errorf("Value is empty")
+}
+
+// Tests valid configurations to create oauth2 clients
+func TestAccMeApiOauth2Client_importBasic(t *testing.T) {
```

Comment:
rename to TestAccMeApiOauth2Client_basic since the import is not the only test step.